src/mono/System.Private.CoreLib/src/System/RuntimeType.Mono.cs

```diff
@@ -2118,15 +2118,15 @@ public override InterfaceMapping GetInterfaceMap([DynamicallyAccessedMembers(Dyn
 
             InterfaceMapping res;
             if (!ifaceType.IsInterface)
-                throw new ArgumentException("Argument must be an interface.", nameof(ifaceType));
+                throw new ArgumentException(SR.Argument_MustBeInterface, nameof(ifaceType));
```

Comment:
```suggestion
                throw new ArgumentException(SR.Arg_MustBeInterface, nameof(interfaceType));
```
Better to use existing message with the same meaning.

Also, the argument name should be updated to match the argument name in the public surface.